### PR TITLE
refactor: Clean up templates and plugin structure

### PR DESCRIPTION
**Pull request by stefanjudis**
This PR cleans up the plugin structure and adds references to the IGSG. There is no change of functionality.

### CODE REVIEW

File: WHATS-NEXT.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# What's next?
-
-A blog on your local machine is nice to have, but a blog on the internet is even better.
-
-## Deploying using now.sh
-
-Change directory to where you installed your application and run:
-
-`npm run deploy`
```
Comment (axe312ger) [inline WHATS-NEXT.md]: this file is shown by the CI after the last step. We can not delete it ;)

File: components/navigation.vue [hunk 2]
```diff
@@ -2,18 +2,12 @@
   <nav role="navigation" class="top-nav">
     <ul class="menu">
       <li v-if="!/\/.*?\/.*/.test(route.path)" v-for="route in this.$router.options.routes">
-        <nuxt-link :to=" { name: route.name }">{{ route.name.charAt(0).toUpperCase() + route.name.slice(1) }}</nuxt-link>
+        <nuxt-link :to="{ name: `${route.name}` }">{{ route.name.charAt(0).toUpperCase() + route.name.slice(1) }}</nuxt-link>
```
Comment (axe312ger) [inline components/navigation.vue]: unnecessary template string

File: lib/config.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+
```
Comment (axe312ger) [inline lib/config.js]: No empty lines at the beginning IMHO

**Comment by stefanjudis** [comment]
@axe312ger Change requests resolved.

File: lib/config.js (hunk 3 above)
Comment (axe312ger) [inline lib/config.js]: 👀 ❓

**Comment by axe312ger** [comment]
Nice, just one thing overlooked.

Also, I would deprecate #7 in favor of https://github.com/contentful/blog-in-5-minutes/pull/14/files#diff-d2ebad225d618e8c69447e93aaa320f8R11

**Comment by stefanjudis** [comment]
@axe312ger yes, that's the  idea. :)

**Pull request merged**